### DISCUSSION TITLE
design mode cannot split vertically

### DISCUSSION THREAD

**Discussion by pietro-coretto** [DeaDBeeF-Player/deadbeef#3063] [General]
Deadbeef version: 1.9.2
OS: Linux Ubuntu 20.04 LTS

I 'm trying to get a layout the one seen below using "design mode" but I cannot split the area vertically so to put cover art on the left and song list on the right. I'd prefer to do this directly over the config file but it is not clerar where to start to leaen the various options 

![deadbeef-theme](https://github.com/DeaDBeeF-Player/deadbeef/assets/71322176/2984d566-50ae-42e9-9548-654301d8dba9)

**Comment by Oleksiy-Yakovenko** [comment]
You need to enter design mode, then right click and choose "replace..." menu, and then choose the layout option that you need. For example, vertical splitter

**Comment by pietro-coretto** [comment]
thanks... I did it, and it's now almost as I wanted to be. Could you suggest how to format the string indicated with right arrow? The title of the grouped list, I want it like "Artist - Album - Year "... I couldn't locate where to change it.

![2024-03-30-184723-screenshot](https://github.com/DeaDBeeF-Player/deadbeef/assets/71322176/ac055af1-6927-4a3e-a879-80b16ff4b560)

**Comment by Oleksiy-Yakovenko** [comment]
It looks like one of the default grouping presets. You can enable grouping by right clicking on playlist column headers, and select Group By -> Artist/Date/Album
